mounts-template.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file should not be checked in, otherwise you'll change the default behavior. You already have mounts-template.json.example